diesel_tests/Cargo.toml

```diff
@@ -13,11 +13,12 @@ dotenv = "0.8.0"
 [dependencies]
 assert_matches = "1.0.1"
 chrono = { version = "^0.2.17" }
-diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuid"] }
+diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuid", "serde_json"] }
```

Comment:
I wonder if we should add an `everything` feature so we use that here (which would also reduce churn in the travis scripts)